portugues/colabora/semdiferencas/pentefino/pentefino.md

```diff
@@ -2,4 +2,4 @@ Depois de passar horas analisando e depurando, você
 
 [Descobre um erro de lógica](../../comdiferencas/diff/diff.md)
 
-[Já não consegue raciocinar e fica babando no teclado de tanto sono](../../colaborar.md)
```

Comment:
Well, if I added only new lines would have same results as this file didn't contain a newline at the end of file, and so Git would say the last line was deleted and added again right after.
Because of that auto-merge would have not worked properly anyway.